docs/UserGuide.md

```diff
@@ -114,15 +114,15 @@ Examples:
 *  `edit 1 i/S8340008J p/91234567 e/johndoe@example.com` Edits the NRIC, phone number and email address of the 1st elderly shown in the list to be `91234567` and `johndoe@example.com` respectively.
 *  `edit S6878830G n/Betsy Crower t/` Edits the name of the elderly with NRIC `S6878830G` to be `Betsy Crower` and clears all existing tags.
 
-### Locating elderly by name or nric: `find`
+### Locating elderly by name and/or nric: `find`
 
 Finds all elderly whose names or nric contain any of the given keywords.
 
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
 * The search is case-insensitive. e.g `hans` will match `Hans`
 * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name or nric is searched.
+* Only the name and/or nric is searched.
```

Comment:
i think locating elderly by name and/or nric sounds ok cuz that is like the intention of the command

but the `Only the name and/or nric is searched.` more has to do with what the code actually does, and it searches both name AND nric, not really and/or, maybe `only the name and nric are searched` sounds better